src/compiler/transformers/test/proxy-custom-element-function.spec.ts

```diff
@@ -97,7 +110,7 @@ describe('proxy-custom-element-function', () => {
 
         expect(formatCode(transpiledModule.outputText)).toContain(
           formatCode(
-            `export const foo = 'hello world!', ${componentClassName} = /*@__PURE__*/ __stencil_proxyCustomElement(class ${componentClassName} extends HTMLElement {}, true);`
+            `const foo = 'hello world!', ${componentClassName} = /*@__PURE__*/ __stencil_proxyCustomElement(class ${componentClassName} extends HTMLElement {}, true);`
```

Comment:
just curious, and I'm probably just not fully following the logic in the diff here, but why did this test change? is there an `export ${componentClassName}` elsewhere in `transpiledModule.outputText`?

Reply:
I don't think I explained this part well in my novella of a PR summary - this is due to the changes here - https://github.com/ionic-team/stencil/pull/4447/files#diff-5a3f4b576667cc1940602dba4a00df74180725754979fb8a88e1e51e3761d87eL87

Previously we were always assuming we should just tack on an `export` keyword, which I think was actually a really subtle bug that never managed to bite us until now. With the linked change, we just carry the modifiers over from the variable statement and let the compiler determine where those should be used elsewhere

Reply:
That seems like what should have been the behavior all along, so I think that makes sense 👍